apps/web/src/api/document/create/types.ts

```diff
@@ -1,7 +1,7 @@
 import { z } from 'zod';
 
 export const ZCreateDocumentRequestSchema = z.object({
-  file: z.instanceof(File),
+  file: z.custom<File>((v) => v instanceof File),
```

Comment:
fixes this error

![Screenshot 2023-08-28 16:31:56](https://github.com/documenso/documenso/assets/84864519/58c27ca0-2659-4a47-8d7a-a699e0b90d5c)